### PR TITLE
New version: yt-dlp.yt-dlp.nightly version 2024.02.03.232712

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137784)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-02-03 20:26:13.290 [FAIL] WindowsPackageManager.dll!00007FF8A66FBDD2: ReturnHr(1) tid(1e1c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FF8A66E203A: (caller: 00007FF8A65C7669) Exception(1) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:13.290 [FAIL] WindowsPackageManager.dll!00007FF8A6583AF1: LogHr(1) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:13.291 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(53)\WindowsPackageManager.dll!00007FF8A6707301: (caller: 00007FF8A65D04FE) LogHr(2) tid(1e1c) 80190193 Forbidden (403).     Msg:[winrt::hresult_error: Forbidden (403).]  2024-02-03 20:26:13.291 [REPO] Source add/update failed, waiting a bit and retrying: winget 2024-02-03 20:26:15.321 [FAIL] WindowsPackageManager.dll!00007FF8A66FBDD2: ReturnHr(2) tid(1e1c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FF8A66E203A: (caller: 00007FF8A65C7669) Exception(2) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.321 [FAIL] WindowsPackageManager.dll!00007FF8A6583AF1: LogHr(3) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.322 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(540)\WindowsPackageManager.dll!00007FF8A67068E3: (caller: 00007FF8A64C8B8D) LogHr(4) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.322 [REPO] Failed to update source: winget 2024-02-03 20:26:15.502 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FF8A6706B53: (caller: 00007FF8A64C8B8D) LogHr(5) tid(1e1c) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8A6619298: (caller: 00007FF8A66D24F9) Exception(3) tid(1e1c) 8A15005E ]  2024-02-03 20:26:15.502 [REPO] Failed to open available source: msstore 2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FF8A65E453A: (caller: 00007FF8A65D0B9E) Exception(4) tid(1e1c) 8A15000F  2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FF8A6706B53: (caller: 00007FF8A64C8B8D) LogHr(6) tid(1e1c) 8A15000F     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FF8A65E453A: (caller: 00007FF8A65D0B9E) Exception(4) tid(1e1c) 8A15000F ]  2024-02-03 20:26:15.519 [REPO] Failed to open available source: winget 2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(571)\WindowsPackageManager.dll!00007FF8A65D10DB: (caller: 00007FF8A64C8B8D) Exception(5) tid(1e1c) 8A15004B  2024-02-03 20:26:33.769 [CLI ] ShellExecute installer failed: 2 Installation failed with exit code -1978334972

(Automated response - build 695.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-02-03 20:26:13.290 [FAIL] WindowsPackageManager.dll!00007FF8A66FBDD2: ReturnHr(1) tid(1e1c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FF8A66E203A: (caller: 00007FF8A65C7669) Exception(1) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:13.290 [FAIL] WindowsPackageManager.dll!00007FF8A6583AF1: LogHr(1) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:13.291 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(53)\WindowsPackageManager.dll!00007FF8A6707301: (caller: 00007FF8A65D04FE) LogHr(2) tid(1e1c) 80190193 Forbidden (403).     Msg:[winrt::hresult_error: Forbidden (403).]  2024-02-03 20:26:13.291 [REPO] Source add/update failed, waiting a bit and retrying: winget 2024-02-03 20:26:15.321 [FAIL] WindowsPackageManager.dll!00007FF8A66FBDD2: ReturnHr(2) tid(1e1c) 80190193 Forbidden (403).     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\HttpStream\HttpClientWrapper.cpp(50)\WindowsPackageManager.dll!00007FF8A66E203A: (caller: 00007FF8A65C7669) Exception(2) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.321 [FAIL] WindowsPackageManager.dll!00007FF8A6583AF1: LogHr(3) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.322 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(540)\WindowsPackageManager.dll!00007FF8A67068E3: (caller: 00007FF8A64C8B8D) LogHr(4) tid(1e1c) 80190193 Forbidden (403). 2024-02-03 20:26:15.322 [REPO] Failed to update source: winget 2024-02-03 20:26:15.502 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FF8A6706B53: (caller: 00007FF8A64C8B8D) LogHr(5) tid(1e1c) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FF8A6619298: (caller: 00007FF8A66D24F9) Exception(3) tid(1e1c) 8A15005E ]  2024-02-03 20:26:15.502 [REPO] Failed to open available source: msstore 2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FF8A65E453A: (caller: 00007FF8A65D0B9E) Exception(4) tid(1e1c) 8A15000F  2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(564)\WindowsPackageManager.dll!00007FF8A6706B53: (caller: 00007FF8A64C8B8D) LogHr(6) tid(1e1c) 8A15000F     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Microsoft\PreIndexedPackageSourceFactory.cpp(205)\WindowsPackageManager.dll!00007FF8A65E453A: (caller: 00007FF8A65D0B9E) Exception(4) tid(1e1c) 8A15000F ]  2024-02-03 20:26:15.519 [REPO] Failed to open available source: winget 2024-02-03 20:26:15.519 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(571)\WindowsPackageManager.dll!00007FF8A65D10DB: (caller: 00007FF8A64C8B8D) Exception(5) tid(1e1c) 8A15004B  2024-02-03 20:26:33.769 [CLI ] ShellExecute installer failed: 2 Installation failed with exit code -1978334972

(Automated response - build 695.)

File: manifests/y/yt-dlp/yt-dlp/nightly/2024.02.03.232712/yt-dlp.yt-dlp.nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: yt-dlp.yt-dlp.nightly
+PackageVersion: 2024.02.03.232712
+InstallerLocale: en-US
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/y/yt-dlp/yt-dlp/nightly/2024.02.03.232712/yt-dlp.yt-dlp.nightly.installer.yaml:7]:
```suggestion
InstallerType: `portable`
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**